src/components/manifold-subscription-details/components/PlanSelector.tsx

```diff
@@ -7,9 +7,47 @@ import {
   PlanFixedFeatureEdge,
   PlanMeteredFeatureEdge,
   PlanConfigurableFeatureEdge,
+  UpdateSubscriptionMutation,
+  UpdateSubscriptionMutationVariables,
 } from '../../../types/graphql';
 import state from '../store';
```

Comment:
None of these changed are important. Take a look at the `subscription-details` directory instead.